wiki_articles/learn_cpp.md

```diff
@@ -1,27 +1,30 @@
 # How to Learn C++ Programming
 
-Everyone learns differently, but the best way is to learn by doing:
-Working on personal projects, solving programming challenges, etc.
-Here are some resources to accompany you:
+We generally recommend a good book to learn the necessary fundamentals:
+- [Murach’s C++ Programming](https://www.murach.com/shop/murach-s-c-programming-2nd-edition-detail)
+- [Programming: Principles and Practice Using C++](https://stroustrup.com/programming.html)
+- [learncpp.com](https://www.learncpp.com/)
```

Comment:
Would you be opposed to a note here that learncpp is really geared towards people who have prior programming experience?

Reply:
Is it? That's not the impression I got. If it is then I think we should probably not link to it here. At least if this article is mean for beginner…

Reply:
My main concern with learn-cpp is that it takes over 100 chapters to introduce loops. It's a very bottom up approach which can be great to get a solid foundation for the language but takes too long to introduce core concepts and if you're new to programming in general you probably don't know how to jump around to the important parts.

Reply:
Yeah, I'm not a big fan of learncpp.com in general. That's why I put those books at the top. I'm not sure what would be a better free tutorial to link.

Reply:
I'm not sure of one either, it's a bit unfortunate there isn't a better one. I'd include a note about learncpp similar to the one we currently have if you feel the space can be afforded.

Reply:
I don't think a note is necessary. It's the least worst free option I'm aware of. It is what it is. If we knew a better option, that would be there instead. I think we should either include resources that we can actually recommend, or not include them at all. I don't think things that need an asterisk attached should be in the shortlist of recommendations.